model-engine/model_engine_server/inference/batch_inference/vllm_batch.py

```diff
@@ -10,6 +10,7 @@
 
 import boto3
 import smart_open
+from datadog import statsd
```

Comment:
This import looks like its no longer required, can we remove the datadog dependency in the requirements.txt as well?

Reply:
yeah that's probably also causing lint error